lib/api/channel.js

```diff
@@ -87,7 +87,7 @@ class Channel extends ApiResource {
       /**
        * @var {{channels, paging}} res
        */
-      this.context.authRequest('get', `/users/self/channels.json?pagesize=${pageSize}&page=${page}`)
+      this.context.authRequest('get', `/users/self/channels.json?pagesize=${pageSize}&page=${page}&p=${page}`)
```

Comment:
It seems the latest version of the API uses new pagination parameters. The param `page=${page}` is no longer needed. Please remove it. Additionally, these params should be  refactored to use `qs`:

```
qs.stringify({pagesize, p: page})
```